.github/workflows/test-vm.yml

```diff
@@ -17,7 +17,7 @@ jobs:
           - ubuntu-2204
           - fedora-39
     container: 
-      image: "manjusakalza/bpftime-base-image:${{matrix.container}}"
+      image: "hp77creator/bpftime-base-image:${{matrix.container}}"
```

Comment:
What about getting these images to account eunomia-bpf? Do you have permission to do that?

Reply:
for the ghci images, will have to configure i guess, For the time being, I focussed more on making the existing setup pass. We can create an issue to migrate these to eunomia ones. Works @Officeyutong ?

Reply:
#341 addresses this. You can add more requirements on the issue if I am missing something.